### DISCUSSION TITLE
Deploying amplify code

### DISCUSSION THREAD

**Discussion by gr-qft** [vercel/next.js#20506] [Help]
Hello friends, 

Hope somebody will be able to help me with the following. 

I am using amplify to write code and would like to deploy using vercel. The problem is the file `aws-exports.js` generated by amplify that's included automatically in .gitignore. Vercel cannot build my project without that file, and so I removed it from .gitignore. Is this ok? And if not, what's the alternative? 

Thank you!

**Comment by bernardobelchior** [comment]
I've spent some hours on this. From what I've seen there's no easy answer. 
I've tried running `amplify checkout dev` in the `npm run build` script, but it didn't work.
Since I'm the only developer, I've decided to remove it from the .gitignore, however I don't know how that will work with teams of multiple developers.

**Reply by gr-qft** [reply]
I see, thanks for sharing your thought. I ended up trying https://www.serverless.com/ and the problem became irrelevant, and the app still seemed deployed on vercel.
